dot/digest/digest.go

```diff
@@ -103,9 +103,14 @@ func (h *Handler) toConsensusDigests(scaleVaryingTypes []scale.VaryingDataType)
 	consensusDigests := make([]types.ConsensusDigest, 0, len(scaleVaryingTypes))
 
 	for _, d := range scaleVaryingTypes {
-		digest, ok := d.Value().(types.ConsensusDigest)
+		digestValue, err := d.Value()
+		if err != nil {
+			h.logger.Debug(err.Error())
+			continue
+		}
+		digest, ok := digestValue.(types.ConsensusDigest)
 		if !ok {
-			h.logger.Debugf("digest type not supported: %T", d.Value())
+			h.logger.Debugf("digest type not supported: %T", digestValue)
```

Comment:
nit a tiny bit out of scope: shouldn't we log this at the error level? Or even panic? :thinking:

Reply:
updated!